tests/conftest.py

```diff
@@ -12,6 +12,7 @@ def test_paths():
     test_paths.safe = f"{test_path}/data/S1A_IW_SLC__1SDV_20200511T135117_20200511T135144_032518_03C421_7768.zip"
     test_paths.orbit_dir = f"{test_path}/data/orbits"
     test_paths.orbit_file = "S1A_OPER_AUX_POEORB_OPOD_20210318T120818_V20200510T225942_20200512T005942.EOF"
+    test_paths.dem_file = f"{test_path}/data/dummy_dem.tif"
```

Comment:
If there's no meaningful data other than EPSG in the DEM file, perhaps it can generated on the fly in the conftest as a session fixture?

Reply:
yeah that sounds like a good solution too
I'm breaking off the unit test to here https://github.com/opera-adt/s1-reader/pull/108 so that Virginia can proceed with COMPASS SET testing